CHANGELOG.md

```diff
@@ -1,32 +1,218 @@
 # Changelog
 
+## Cargo 1.76 (2024-02-08)
+[6790a512...HEAD](https://github.com/rust-lang/cargo/compare/6790a512...HEAD)
+
+### Added
+
+### Changed
+
+### Fixed
+
+- Fixed `--quiet` being used with nested subcommands.
```

Comment:
I think this fix is more theoretical at this point and so more of a refactor rather than a fix for things people are running into.

Thoughts?